### DISCUSSION TITLE
php artisan native:build

### DISCUSSION THREAD

**Discussion by ikasv** [NativePHP/laravel#223] [Q&A] [answered]
Where I can get the exported file ?

**Comment by mohamedds-12** [comment]
Same thing.... I can't seem to find it anywhere !!

**Comment by NBA707** [comment, marked as answer]
It is always placed in the dist folder under the project's folder.   If it's not there then the build failed for some reason.

**Reply by NBA707** [reply]
![Screenshot 2023-11-07 at 6 34 25 AM](https://github.com/NativePHP/laravel/assets/52007275/c7e5d806-91e8-4449-8a94-db33b844f7a6)
